packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -786,6 +786,7 @@ export const components = ({
 
               ...(ownerState.size === "small" && {
                 padding: odysseyTokens.Spacing2,
+                width: odysseyTokens.Spacing6,
```

Comment:
What is this for? Seems like a pretty broad application. We likely do not want to be setting a `width` for any button, unless it's `100%`

Reply:
Icon buttons should always be square, but the `small` icon button was a hair wider than tall; this ensures the small icon button is always the right size.

Reply:
What is causing them to be wider though? I suspect that the icon is now probably off-center right?

Reply:
Is the icon that is passed in not square? I'd rather see us manipulate the icon size and let that continue to set the button size than explicitly set a width on a button

Reply:
Yeah, that's fair. Let's remove the width constraint for now and figure out if we need to adjust the button padding/width more holistically as a separate PR.